api/pom.xml

```diff
@@ -12,10 +12,14 @@
   <name>SmallRye Reactive Messaging API</name>
 
   <dependencies>
-    <dependency>
-      <groupId>org.eclipse.microprofile.reactive.messaging</groupId>
-      <artifactId>microprofile-reactive-messaging-api</artifactId>
-    </dependency>
+    <!--
```

Comment:
Maybe leave the dependency there but commented out?

Reply:
Yes, I was wondering about that, probably a good idea.

Reply:
Done.